bigquery/docs/index.rst

```diff
@@ -1,57 +1,7 @@
-BigQuery
-========
-
-.. toctree::
-  :maxdepth: 2
-  :hidden:
-
-  reference
-  dbapi
-
-.. contents:: :local:
-
-Installation
-------------
-
-Install the ``google-cloud-bigquery`` library using ``pip``:
-
-.. code-block:: console
-
-    $ pip install google-cloud-bigquery
-
-.. note::
-
-    This library changed significantly before the 1.0.0 release, especially
-    between version 0.27 and 0.28. See `Migrating from the BigQuery Python
-    client library version 0.27
-    <https://cloud.google.com/bigquery/docs/python-client-migration>`__ for
-    instructions on how to migrated your code to the most recent version of
-    this library.
-
-Authentication / Configuration
-------------------------------
-
-- Use :class:`Client <google.cloud.bigquery.client.Client>` objects to configure
-  your applications.
-
-- :class:`Client <google.cloud.bigquery.client.Client>` objects hold both a ``project``
-  and an authenticated connection to the BigQuery service.
-
-- The authentication credentials can be implicitly determined from the
-  environment or directly via
-  :meth:`from_service_account_json <google.cloud.bigquery.client.Client.from_service_account_json>`
-  and
-  :meth:`from_service_account_p12 <google.cloud.bigquery.client.Client.from_service_account_p12>`.
-
-- After setting :envvar:`GOOGLE_APPLICATION_CREDENTIALS` and
-  :envvar:`GOOGLE_CLOUD_PROJECT` environment variables, create an instance of
-  :class:`Client <google.cloud.bigquery.client.Client>`.
-
-  .. code-block:: python
-
-     >>> from google.cloud import bigquery
-     >>> client = bigquery.Client()
+.. include:: /../bigquery/README.rst
```

Comment:
Can we add a redirect from the usage page to the index page so that we don't end up with broken links on cloud.google.com?

Reply:
Oops, that doesn't work:  it breaks Sphinx.

Reply:
"This time for sure, Bullwinkle!"  15610f2